cmake/platform/windows.cmake

```diff
@@ -88,3 +88,41 @@ if(MSVC)
   # https://blogs.msdn.microsoft.com/vcblog/2018/04/09/msvc-now-correctly-reports-__cplusplus/
   add_compile_options(/Zc:__cplusplus)
 endif()
+
+#
+# Helper function to add Python executable wrapper on Windows due to lack of shebang support
+#
+# Code courtesy of Yujin Robot.  Derived from https://github.com/ros-windows/win_ros
+#
+
+#
+# Add Python executable wrapper around Python scripts on Windows.
+# Python scripts with (or without) .py extension are not executable on Windows due to lack of shebang support
+#
+# :param script: Python script name that needs a wrapper
+# :type script: string
+# :param target: build target name
+# :type target: string
+#
+# @public
+#
+function(add_python_executable_helper)
```

Comment:
The `_helper` suffix doesn't add anything to the naming of this function. How about just `add_python_executable`?